lib/iris/_data_manager.py

```diff
@@ -221,6 +221,10 @@ def _deepcopy(self, memo, data=None, realised_dtype=None):
                 # If the replacement data is valid, then use it but
                 # without copying it.
 
+            if isinstance(realised_dtype, six.string_types) and \
```

Comment:
See [this](https://github.com/SciTools/iris/pull/2492#discussion_r112475921) comment response ...